README.md

```diff
@@ -2,23 +2,42 @@
 
 ![Screenshot from 2023-03-18 15-29-05](https://user-images.githubusercontent.com/8228060/226112255-fe2d4cdc-193e-4c23-8a36-3d8f60baaf03.png)
 
+This project contains an awesome application called `QuickPizza` and also demonstrates how to use the different features of k6. 
+
+You'll find sample tests about:
+
+- Basic HTTP load test
+- Stages and lifecycles in k6
+- Scenarios in k6
+- Browser testing via k6 browser
+- Hybrid performance test (HTTP + xk6-disruptor, browser + xk6-disruptor)
+- and many more!
+
+This can be used for demonstrations or workshops.
+
+## Requirements
+
+- [Docker](https://docs.docker.com/get-docker/)
+- [Minikube](https://minikube.sigs.k8s.io/docs/start/)
```

Comment:
Can we note here that the Kubernetes setup is only required if you want to test the disruptor? :eyes: 

All other k6 scripts will work with plain Docker.

Reply:
Resolve in https://github.com/grafana/quickpizza/pull/4/commits/1fc71ef5f8e0ce58240c7ec146253e28dc55c078

Reply:
Sweet!